comictaggerlib/taggerwindow.py

```diff
@@ -1022,11 +1022,12 @@ def query_online(self, autoselect: bool = False, literal: bool = False) -> None:
 
         issue_number = str(self.leIssueNum.text()).strip()
 
-        if autoselect and issue_number == "":
+        # This is also done in the volumeselectionwindow?
+        """if autoselect and issue_number == "":
             QtWidgets.QMessageBox.information(
                 self, "Automatic Identify Search", "Can't auto-identify without an issue number (yet!)"
             )
-            return
+            return"""
```

Comment:
This is here in order to fail early. We don't want to do an entire search with possibly wrong data just to say it's pointless.